pkg/chartverifier/checks/registry.go

```diff
@@ -26,6 +26,28 @@ type Result struct {
 	Reason string
 }
 
+func NewResult(outcome bool, reason string) Result {
+	result := Result{}
+	result.Ok = outcome
+	result.Reason = reason
+	return result
```

Comment:
What is the benefit of having this constructor here versus simple `Result{Ok: true, Reason: "foo"}`as before? I think we can revert it?

Reply:
This is all going to change as part of creating the report schema, so OK if I defer and roll updates into that?